functions/go/set-namespace/main.go

```diff
@@ -142,16 +208,15 @@ Example:
 
 To add a namespace 'color' to Deployment resource only:
 
-apiVersion: v1
-kind: ConfigMap
+apiVersion: kpt.dev/v1beta1
+kind: SetNamespaceConfig
 metadata:
   name: my-config
-data:
-  namespace: color
-  fieldSpecs:
-    - path: metadata/namespace
-      kind: Deployment
-      create: true
+namespace: color
+fieldSpecs:
+- path: metadata/namespace
+  kind: Deployment
```

Comment:
Does the default config have this for Deployment?

Reply:
This is used to *add a namespace 'color' to Deployment resource **only***

Reply:
Got it. It overrides the default config. I somehow thought it got appended to the default config.

We should call it out explicitly for the advanced way. IMO we should also provide a pointer to the default config, since user can copy them if needed.

Reply:
I am thinking we should append the fieldSpec now to keep consistent. Will update it.